src/hellpers/main.hellpers.js

```diff
@@ -6,9 +6,64 @@ const getRequestBody = req => new Promise((res, rej) => {
     .on('data', chunk => buffer += chunk)
     .on('error', err => rej(err))
     .on('end', () => res(buffer))
-})
+});
+
+const getResponseBody = data => new Promise(res => {
```

Comment:
Are you using this function for products filtering? If so, it would be better to call it `filter....`. Get is a little confusing here